selectrum.el

```diff
@@ -979,18 +979,18 @@ The specific details of the formatting are determined by
                                (minibuffer-contents)
                                lines)))
                  (match
-                  (concat
-                   (propertize
-                    (propertize newline/display 'face newline/face)
-                    'selectrum-candidate-display-prefix
-                    (number-to-string (1- len)))
+                  (propertize
                    (replace-regexp-in-string
                     "[ \t][ \t]+"
                     (propertize whitespace/display 'face whitespace/face)
                     (if (string-empty-p (minibuffer-contents))
                         ""
                       ;; Show first matched line.
-                      (or fmatch "")) 'fixed-case 'literal)))
+                      (or fmatch "")) 'fixed-case 'literal)
+                   'selectrum-candidate-display-prefix
+                   (propertize (concat (number-to-string (1- len))
+                                       newline/display ":")
+                               'face newline/face)))
```

Comment:
```suggestion
                   (propertize (format "(%d lines)"
                                       (1- len))
                               'face newline/face)))
```

What do you think of this? I feel like it is more unambiguous, without being too bulky:

![image](https://user-images.githubusercontent.com/6559064/103143628-f4406400-46ce-11eb-8f71-ce8c447387f1.png)

Reply:
Much better!